applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -386,12 +386,6 @@ You can add the following optional configurations to configure the heap or to pr
 * :kconfig:`CONFIG_PDN_DEFAULTS_OVERRIDE` - Used for manual configuration of the APN. Set the option to ``y`` to override the default PDP context configuration.
 * :kconfig:`CONFIG_PDN_DEFAULT_APN` - Used for manual configuration of the APN. An example is ``apn.example.com``.
 
-The application supports Assisted GPS.
```

Comment:
Why is this removed? The application still supports A-GPS.

Reply:
The section is called "Optional library configurations" and A-GPS is enabled by default, so I didn't see this as necessary anymore.

Reply:
But `CONFIG_NRF_CLOUD_AGPS` is still an optional setting, and needs extra set up on nRF Cloud side AFAIK: in order to use nRF Cloud A-GPS must be activated manually).